CHANGELOG/CHANGELOG-2.5.md

```diff
@@ -0,0 +1,80 @@
+# Changelog 2.5
+
+## 2.5.0 - 2025-11-28
+
+### Added
+
+- Support `nftables` in Node host network for AntreaProxy. ([#7545](https://github.com/antrea-io/antrea/pull/7545), [@hongliangl])
+- Add support for Antrea Egress in hybrid mode. ([#7239](https://github.com/antrea-io/antrea/pull/7239), [@hongliangl])
+- Add direction flag to antctl packetcapture command. ([#7195](https://github.com/antrea-io/antrea/pull/7195), [@sratslla])
+- Add IPv6 support for PacketCapture feature, including ICMPv6 handling and new CLI filter options. ([#7385](https://github.com/antrea-io/antrea/pull/7385), [@harshgdev])
+- Enhance PacketCapture to support matching arbitrary source or destination. ([#7215](https://github.com/antrea-io/antrea/pull/7215), [@harshgdev])
+- Enhance PacketCapture to also capture packets at the destination Pod, in addition to the source. ([#7289](https://github.com/antrea-io/antrea/pull/7289), [@harshgdev])
+- Add a new `FlowExporterTarget` CRD to enable defining multiple flow exporter targets with different rules and protocols. ([#7494](https://github.com/antrea-io/antrea/pull/7494), [@andrew-su])
```

Comment:
don't include this one: we have merged the CRD definition but the implementation will not be added until Antrea v2.6, so the CRD is not usable for now